### ISSUE TITLE
News update from 0.6.0 to 0.7.0 for issue 745 examples

### ISSUE THREAD

**Issue by bms63** [pharmaverse/admiral#932]
I merged in #745 and realized later that it is listed in 0.6.0 news release, but it actually did not make it in.  So #745 will be released into 0.7.0.

Could you update the news sections for 0.7.0 once #900 is merged into `devel `branch.  #900 is a skeleton news for the next release. 
 Sorry about that!

**Comment by bms63** [comment]
Hey Tom,

I'm going to help Ojesh make this update as a demo of the Pull Request process.

**Comment by tomratford** [comment]
Thanks Ben! 👍

**Comment by bms63** [comment]
@upadhyor Ojesh just for your reference this was the issue we were tackling today.  Was `renv::restore() `successful?

**Comment by upadhyor** [comment]
@bms63 renv::restore() was successful.  I made sure to decrease the RAM and CPU and apply the change before logging off from RStudio Cloud.  Thank you for your support to help me get started!
